frontend/src/index.css

```diff
@@ -265,8 +264,8 @@ body, html {
 	font-size: 48px;
 }
 
-.backpack .unit-avatar,
-.board-cell .unit-avatar {
-	cursor: pointer;
+.unit-avatar-draggable {
+	cursor: move;
```

Comment:
Am I not aware of some css mysteriousness? Is the one property not overriding the other?

Reply:
not all browser support grab, this is just a default value for them

Reply:
Ah, alright.